### PR TITLE
Prod&Dev Environment

### PR DESCRIPTION
**Pull request by mishagitwork**
Чекаем, ругаемся, апруваем. Кому что по душе. 
docker-compose -f docker-compose.dev.yml up --build   - сборка dev среды
docker-compose -f docker-compose.prod.yml up --build   - сборка prod среды
Заменяем up --build на down для остановки и удаления всех контейнеров

### CODE REVIEW

File: docker-compose.dev.yml [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
 services:
-  db:
+  db_dev:
     image: postgres
     restart: always
     environment:
       POSTGRES_PASSWORD: trytofindme
     volumes:
-      - ./db:/var/lib/postgresql/data
+      - ./db:/var/lib/postgresql/data_dev
```
Comment (i-barysiuk) [inline docker-compose.dev.yml]: It is not working

DATA_IN_LOCAL_DIRECTORY:DATA_IN_CONTAINER

Postrges not read data from custom directory "data_dev"

File: npm-debug.log [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+9 error npm  v3.5.2
+10 error path /home/pc/BSTU/package.json
+11 error code ENOENT
+12 error errno -2
+13 error syscall open
+14 error enoent ENOENT: no such file or directory, open '/home/pc/BSTU/package.json'
+15 error enoent ENOENT: no such file or directory, open '/home/pc/BSTU/package.json'
+15 error enoent This is most likely not a problem with npm itself
+15 error enoent and is related to npm not being able to find a file.
+16 verbose exit [ -2, true ]
```
Comment (i-barysiuk) [inline npm-debug.log]: Such file need to add in gitignore

File: docker-compose.prod.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+version: "3.1"
+
+services:
+  db_prod:
+    image: postgres
+    restart: always
+    environment:
+      POSTGRES_PASSWORD: trytofindme
+    volumes:
+      - ./db:/var/lib/postgresql/data_prod
```
Comment (i-barysiuk) [inline docker-compose.prod.yml]: see above

File: docker-compose.prod.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+  adminer_prod:
+    image: adminer
+    restart: always
+    ports:
+      - 8070:8080
+
+  app_prod:
+    build: ./app
```
Comment (i-barysiuk) [inline docker-compose.prod.yml]: we speak about creating different Dockerfile fot prod and DEV
Where are they?

**Comment by i-barysiuk** [comment]
Also please add info about starting project in readme.

File: docker-compose.dev.yml (hunk 1 above)
Comment (mishagitwork) [reply docker-compose.dev.yml]: Will it work if I make it so?
-./db:/var/lib/postgresql/db_dev/data

Comment (i-barysiuk) [reply docker-compose.dev.yml]: NO
You need to change your local path not in container

Comment (mishagitwork) [reply docker-compose.dev.yml]: -./db/db_dev:/var/lib/postgresq/data?

Comment (i-barysiuk) [reply docker-compose.dev.yml]: Yes)

File: docker-compose.prod.yml (hunk 3 above)
Comment (mishagitwork) [reply docker-compose.prod.yml]: изображение

Comment (mishagitwork) [reply docker-compose.prod.yml]: When I use this way.
app_dev:
    build: ./app/dev.Dockerfile
    restart: always
    ports:
      - 8091:3000

Comment (i-barysiuk) [reply docker-compose.prod.yml]: https://docs.docker.com/compose/compose-file/
read about context in docker-compose for custom dockerfile name

Comment (mishagitwork) [reply docker-compose.prod.yml]: It doesn't work too... 
  app_dev:
    build:
        context: ./app
        dockerfile: dev.Dockerfile
    restart: always
    ports:
      - 8091:3000

Comment (i-barysiuk) [reply docker-compose.prod.yml]: Are you create dev.Dockerfile in ./app?

**Pull request merged**